website/blog/2022-08-01-aluminium.md

```diff
@@ -0,0 +1,355 @@
+---
+author: Rikito Taniguchi
+title: Metals v0.11.8 - Aluminium
+authorURL: https://twitter.com/<login>
+authorImageURL: https://github.com/<login>.png
+---
+
+We're happy to announce the release of Metals v0.11.8, which brings a number of improvements for both Scala 2 and Scala3.
+
+<table>
+<tbody>
+  <tr>
+    <td>Commits since last release</td>
+    <td align="center">78</td>
+  </tr>
+  <tr>
+    <td>Merged PRs</td>
+    <td align="center">74</td>
+  </tr>
+    <tr>
+    <td>Contributors</td>
+    <td align="center">14</td>
+  </tr>
+  <tr>
+    <td>Closed issues</td>
+    <td align="center"></td>
```

Comment:
Realized you filled this in 0.11.7 release note https://github.com/scalameta/metals/blob/d2ab724848b0c8b5c066694e5cf431c809715d34/website/blog/2022-07-04-aluminium.md?plain=1#L25-L28 how did you count it? count by hand? or search issues that are closed since the last release?

Reply:
it's done manually. you need to go through closed issues since prev release, set their milestone as 0.11.8 and then you will see the amount here - https://github.com/scalameta/metals/milestone/52

Reply:
Thanks, I checked the closed issues by https://github.com/scalameta/metals/issues?q=is%3Aissue+closed%3A%3E2022-07-04+ and added them to 0.11.8 milestone

Reply:
I wonder if adding closed issues/PRs can't be automated (probably can)